prov/efa/src/efa_user_info.c

```diff
@@ -354,6 +355,31 @@ int efa_user_info_alter_rxr(struct fi_info *info, const struct fi_info *hints)
 
 			info->domain_attr->mr_mode |= FI_MR_HMEM;
 
+			/* TODO Loop over every HMEM interface and determine if we can support atomics */
+			support_atomic = true;
+			if (hmem_ops[FI_HMEM_CUDA].initialized && cuda_get_xfer_setting() != CUDA_XFER_ENABLED) {
+				/*
+				  HMEM_CUDA + ATOMICS require CUDA MEMCPY to be used.
+				  We are making an assumption here that the remote we are performing atomic
```

Comment:
each line of comment should start with `*`.

Reply:
done